### PR TITLE
chore(config): added optimism to config and fixed broken link

### PR DESCRIPTION
**Pull request by crisgarner**
# Description

Added optimism to config and fixed a broken link

## Confirmation

- [x] I have read and understand MACI's [contributor guidelines](https://maci.pse.dev/docs/contributing) and [code of conduct](https://maci.pse.dev/docs/contributing/code-of-conduct).
- [x] I have read and understand MACI's [GitHub processes](https://github.com/privacy-scaling-explorations/maci/discussions/847).
- [x] I have read and understand MACI's [testing guide](https://maci.pse.dev/docs/testing).

### CODE REVIEW

File: website/versioned_docs/version-v2.0_alpha/quick-start/installation.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ pnpm run build
 ```
 
 :::info
-Depending on the number of signups you would like to support, you will be required to re-compile the contracts with a different STATE_TREE_DEPTH parameter. This is by default 10 and can be used for testing purposes. For more information about the parameters please refer to the [circuits page](/docs/developers-references/zk-snark-circuits/setup#Parameters).
+Depending on the number of signups you would like to support, you will be required to re-compile the contracts with a different STATE_TREE_DEPTH parameter. This is by default 10 and can be used for testing purposes. For more information about the parameters please refer to the [circuits page](/docs/v1.2/circuits#parameters).
```
Comment (ctrlc03) [inline website/versioned_docs/version-v2.0_alpha/quick-start/installation.md:72]: why link to v1.2?

Comment (ctrlc03) [reply website/versioned_docs/version-v2.0_alpha/quick-start/installation.md:72]: oh my bad actually I replied in private with the wrong link 🤦 please can you change to this https://maci.pse.dev/docs/developers-references/zk-snark-circuits/processMessages#parameters

**Review by ctrlc03 — APPROVED** [review]
Thanks

File: website/versioned_docs/version-v2.0_alpha/quick-start/installation.md (hunk 1 above)
Comment (crisgarner) [reply website/versioned_docs/version-v2.0_alpha/quick-start/installation.md]: yeah, the link was fine, the issue was with the header, here is the fix

**Pull request merged**